core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -73,8 +73,63 @@ pub struct BandwidthRequestsV1 {
     ProtocolSchema,
 )]
 pub struct BandwidthRequest {
+    /// Requesting bandwidth to this shard.
     pub to_shard: u8,
-    // TODO(bandwidth_scheduler) - store requested bandwidth values inside the BandwidthRequest
+    /// Bitmap which describes what values of bandwidth are requested.
+    pub requested_values_bitmap: BandwidthRequestBitmap,
```

Comment:
nit: It's nice to have the var name and type name be the same - here it would be bandwidth_requests_bitmap - it's easier to read and grep.

Reply:
Hmm then it's gonna be `bandwidth_request.bandwidth_request_bitmap` x.x
I guess it could also be just `bitmap`, but that's not very descriptive.
I dunno, I think I'll keep it as is. It's still possible to find it by grepping for `BandwidthRequestBitmap`.